supervisor/erratum_handler.py

```diff
@@ -131,6 +136,43 @@ def try_to_advance_erratum(self, new_status: ErrataStatus) -> WorkflowResult:
             )
 
         if rule_set.all_ok:
+            if new_status == ErrataStatus.REL_PREP:
+                packages = get_erratum_packages(self.erratum.id)
+
+                for package in packages:
+                    if not erratum_has_magic_string_in_comments(
+                        self.erratum.id, f"jotnar-product-listings-checked({package})"
+                    ):
+                        prev_erratum = get_previous_erratum(self.erratum.id, package)
+                        if prev_erratum:
+                            if errata_have_same_file_lists(
+                                prev_erratum.id, self.erratum.id
+                            ):
+                                erratum_add_comment(
+                                    self.erratum.id,
+                                    f"jotnar-product-listings-checked({package})\n"
+                                    "Match - The file list of this build match the one of the previous build.",
+                                    dry_run=self.dry_run,
+                                )
+                            else:
+                                erratum_add_comment(
+                                    self.erratum.id,
+                                    f"jotnar-product-listings-checked({package})\n"
+                                    "Mismatch - The file list of this build doesn't match the one of the previous build.",
+                                    dry_run=self.dry_run,
+                                )
+
+                                return self.resolve_flag_attention(
```

Comment:
I would not flag_attention here, that means we'll never compare the other packages (until needs-attention is cleared), I would keep a list of the packages with mismatched builds, and flag needs attention at the end.